VelocidiSDK/VelocidiSDKTests/UtilTests.swift

```diff
@@ -20,6 +19,24 @@ class UtilTests: QuickSpec {
 
                 expect(userAgent).to(match(regexStr))
             }
+            
+            if #available(iOS 14, *) { // ios14 is opt-in
+                it("should throw an error when trying to get the IDFA on iOS 14+") {
+                    // this style is necessary because of https://github.com/Quick/Nimble/issues/809
+                    expect(expression: { try VSDKUtil.tryGetIDFA() }).to(throwError { (error: Error) in
+                        expect(error._domain).to(equal("com.velocidi.VSDKTrackingNotAllowedError"))
+                    })
+                }
+            } else { // previous ios version are opt-out
+                it("should successfuly provide the IDFA") {
+                    if #available(iOS 13, *) {
+                        expect(try? VSDKUtil.tryGetIDFA()).to(contain("00000000-0000-0000-0000-000000000000"))
+                    } else {
+                        expect(try? VSDKUtil.tryGetIDFA()).to(match("[A-Z0-9]{8}-[A-Z0-9]{4}-[A-Z0-9]{4}-[A-Z0-9]{4}-[A-Z0-9]{12}"))
+                    }
```

Comment:
I think we can simplify here and use just the regex.
```suggestion
                    expect(try? VSDKUtil.tryGetIDFA()).to(match("[A-Z0-9]{8}-[A-Z0-9]{4}-[A-Z0-9]{4}-[A-Z0-9]{4}-[A-Z0-9]{12}"))
```

Reply:
Sure!